src/SelectField/SelectField.js

```diff
@@ -120,6 +120,8 @@ class SelectField extends Component {
      */
     onChange: PropTypes.func,
     /** @ignore */
+    onClose: PropTypes.func,
```

Comment:
What about adding, documenting and testing a `dropDownMenuProp` property instead of adding a hidden and specific onClose property?